MonoGame.Framework.Content.Pipeline/TargetPlatform.cs

```diff
@@ -85,7 +85,22 @@ public enum TargetPlatform
         /// <summary>
         /// WebAssembly and Bridge.NET
         /// </summary>
-        Web
+        Web,
+
+        /// <summary>
+        /// Windows GDK
+        /// </summary>
+        GamingDesktop,
+
+        /// <summary>
+        /// Xbox One (GDKX) 
+        /// </summary>
+        GamingXboxOne,
+
+        /// <summary>
+        /// Xbox Series
+        /// </summary>
+        GamingScarlett
```

Comment:
@tomspilman thoughts on naming? Otherwise good to me.

Reply:
I would go with `GamingXboxSeries` because we've avoided code names on all the other ones.

Reply:
> But why the `Gaming` prefix?

It was mainly because it's the name of the target in GDK:
![image](https://github.com/user-attachments/assets/3ff3fee7-b2cd-43c7-8c39-df679d25daf6)
I didn't give much thought when naming these but yeah it's a bit weird as a platform name

I will change that for WindowsGDK and XboxSeries and reuse XboxOne if everyone is ok with that?

Reply:
Sounds good to me as suggested, thanks!

Reply:
I agree, we shouldn't use codenames even if they are in the GDK itself.  Better to set our own statement.

So I agree with WindowsGDK and XboxSeries and reuse XboxOne.  Makes more sense and it clearer to normal devs when they see it.